PowerPointLabs/PowerPointLabs/ColorTheme/ThemeManager.cs

```diff
@@ -74,16 +74,8 @@ private ThemeManager()
 
         private List<int> GetDefaultKeys()
         {
-            if (!Globals.ThisAddIn.IsApplicationVersion2013())
-            {
-                return new List<int>() { ColorTheme.COLORFUL };
-            }
-            return new List<int>()
-            {
-                ColorTheme.WHITE,
-                ColorTheme.LIGHT_GRAY,
-                ColorTheme.DARK_GRAY
-            };
+            // Have no default keys so that ApplyTheme() will always get called.
+            return new List<int>();
```

Comment:
could this be done by removing the need to check against default keys rather than checking against an always empty list?